src/routes/components/Accordion.tsx

```diff
@@ -110,22 +126,13 @@ export default function AccordionPage() {
               incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
               exercitation ullamco laboris nisi
             </GoAAccordion>
-
-            <GoAAccordion
-              {...accordionProps}
-              headingContent={<GoABadge type="success" content="Success" />}
-            >
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-              incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-              exercitation ullamco laboris nisi
-            </GoAAccordion>
           </Sandbox>
 
           <h4>Accordion Group</h4>
           <Sandbox
+            skipRender
             note="The open prop on accordion component can be used to control a group of accordions."
```

Comment:
This note isn't very visible especially after a large code snippet. Was this in the design docs?